build_tools/benchmarks/run_benchmarks_on_android.py

```diff
@@ -412,6 +426,16 @@ def main(args):
         verbose=args.verbose,
     )
 
+    # Turn off thermal throttling. CPU frequencies are throttled when the device is warm.
+    adb_execute_as_root(["setprop", "persist.vendor.disable.thermal.control", "1"])
+
+    # Fake screen-on status. Performance may be affected when screen is off.
+    # This only works on later versions of Android. Disable for now.
+    # adb_execute_as_root(["sendhint", "-m", "DISPLAY_INACTIVE", "-e", "0"])
+
+    if is_pixel_device():
```

Comment:
This enables it unconditionally. It should be controlled by `args.pin_cpu_freq`?